# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2021/04
# - add missing fetk include dirs to geoflow's and pb_s_am's cmake.
# - avoid defining __MKL. Otherwise the final binary cannot be linked due
#   to undefined reference dgemm
diff -ru apbs-3.0.0_pbsam168a83e/externals/pb_s_am/cmake/Dependencies.cmake apbs-3.0.0_pbsam168a83e_cmake/externals/pb_s_am/cmake/Dependencies.cmake
--- apbs-3.0.0_pbsam168a83e/externals/pb_s_am/cmake/Dependencies.cmake	2020-05-22 17:18:22.000000000 +0200
+++ apbs-3.0.0_pbsam168a83e_cmake/externals/pb_s_am/cmake/Dependencies.cmake	2021-04-25 15:41:38.462090915 +0200
@@ -1,5 +1,18 @@
 # Required for static linking and exported to PBSAM.cmake
 set(PBSAM_LINKER_LIBS "")
+include_directories(../../externals/fetk/mc/src/aaa_inc)
+include_directories(../../externals/fetk/mc/src/base)
+include_directories(../../externals/fetk/punc/src/aaa_inc)
+include_directories(../../externals/fetk/punc/src/base)
+include_directories(../../externals/fetk/mc/src/bam)
+include_directories(../../externals/fetk/mc/src/aprx)
+include_directories(../../externals/fetk/mc/src/gem)
+include_directories(../../externals/fetk/mc/src/pde)
+include_directories(../../externals/fetk/mc/src/whb)
+include_directories(../../externals/fetk/mc/src/nam)
+include_directories(../../externals/fetk/mc/src/dyn)
+include_directories(../../externals/fetk/mc/src/mcsh)
+set(CMAKE_VERBOSE_MAKEFILE true)
 
 # ---[ BLAS
 if(NOT APPLE)
@@ -11,7 +24,7 @@
     if( OpenBLAS_FOUND)
       include_directories(${OpenBLAS_INCLUDE_DIR})
       list(APPEND PBSAM_LINKER_LIBS ${OpenBLAS_LIB})
-      add_definitions(-D__MKL)
+#      add_definitions(-D__MKL)
       add_definitions(-D__LAU)
     endif()
   endif()
diff -ru apbs-3.0.0_geoflow3dee4ec/externals/geoflow_c/CMakeLists.txt apbs-3.0.0_geoflow3dee4ec_cmake/externals/geoflow_c/CMakeLists.txt
--- apbs-3.0.0_geoflow3dee4ec/externals/geoflow_c/CMakeLists.txt	2020-11-14 01:01:31.000000000 +0100
+++ apbs-3.0.0_geoflow3dee4ec_cmake/externals/geoflow_c/CMakeLists.txt	2021-04-22 19:04:59.364992969 +0200
@@ -1,4 +1,17 @@
 cmake_minimum_required(VERSION 2.8.7)
+include_directories(
+../../externals/fetk/mc/src/aaa_inc
+../../externals/fetk/mc/src/base
+../../externals/fetk/punc/src/aaa_inc
+../../externals/fetk/punc/src/base
+../../externals/fetk/mc/src/bam
+../../externals/fetk/mc/src/aprx
+../../externals/fetk/mc/src/gem
+../../externals/fetk/mc/src/pde
+../../externals/fetk/mc/src/whb
+../../externals/fetk/mc/src/nam
+../../externals/fetk/mc/src/dyn
+../../externals/fetk/mc/src/mcsh)
 
 enable_testing()
 
